### ISSUE TITLE
Feat: ability to remove workflows in team settings

### ISSUE THREAD

**Issue by vikulikov** [tegonhq/tegon#295]

**Comment by AggManik** [comment]
Hey @vikulikov thanks for creating this issue. I'll explain it further, currently user can add any status or reorder statuses in the workflow for each team but cannot remove any workflow once created.

User should be able to remove the workflow too.

Would you like to pick this issue?

**Comment by vikulikov** [comment]
Yes, thank you! This is it. Now, once workflow is created it can not be removed. It would be nice to be able to remove default ones or created by admin/user

**Comment by vikulikov** [comment]
> Would you like to pick this issue?

Sorry. I didn't understand the question at first. Unfortunately, I have almost zero experience with web stack and js/ts languages

**Comment by harshithmullapudi** [comment]
This is in the pipeline will be released this week
